webservices/schemas.py

```diff
@@ -461,14 +461,7 @@ class CandidateHistoryTotalSchema(schemas['CandidateHistorySchema'], schemas['Ca
             'contributor_employer_text',
             'contributor_occupation_text',
         ),
-        'relationships': [
-            Relationship(
-                models.ScheduleA.committee,
-                models.CommitteeHistory.name,
-                'committee_name',
-                1
-            ),
-        ],
+
```

Comment:
This related information is no longer needed?  That is going to have an impact on the downloads of Schedule A info, no more committee information will appear.  Was the intention to not have it included on the Schedule A E-Filing schema instead?